zuul.d/projects.yaml

```diff
@@ -154,6 +154,20 @@
         - kind-integration-test-arm64:
             branches: master
 
+- project:
+    name: containerd/containerd
+    periodic-6/18:
+      jobs:
+        - containerd-build-arm64-periodic:
+            branches: master
+
+- project:
+    name: envoyproxy/envoy
+    periodic-6/18:
+      jobs:
+        - envoy-build-arm64-periodic:
+            branches: master
```

Comment:
As an example, I think we should remove the jobs define and playbooks in OpenLab side, reasonable step should be:
1. Merge job define and playbook in project side (forked containerd and envoy, as an example, official projects in the future) to **enable PR triggered job**
2. Just refer the merged job name in periodic pipeline of OpenLab side to **enable periodic job**

Reply:
LGTM